blockstore/src/main/java/brooklyn/location/blockstore/AbstractVolumeManager.java

```diff
@@ -73,58 +75,67 @@ public MountedBlockDevice attachAndMountVolume(JcloudsMachineLocation machine, B
     @Override
     public void createFilesystem(AttachedBlockDevice attachedDevice, FilesystemOptions filesystemOptions) {
         JcloudsMachineLocation machine = attachedDevice.getMachine();
-        if (!(machine instanceof SshMachineLocation)) {
-            throw new IllegalStateException("Cannot create filesystem for "+machine+" of type "+machine.getClass().getName()+"; expected "+SshMachineLocation.class.getSimpleName());
```

Comment:
Should we still throw this in an else block (if it's not `SshMacheLocation` or `WinRmMachineLocation`)?